test/e2e/dynamic_resource_e2e_test.go

```diff
@@ -1,105 +1,163 @@
 package e2e
 
 import (
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorclient"
+	"github.com/stretchr/testify/require"
+	"k8s.io/client-go/dynamic"
 	"strings"
 
-	"github.com/stretchr/testify/require"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 
 	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/registry"
 	"github.com/operator-framework/operator-lifecycle-manager/test/e2e/ctx"
 )
 
-var _ = Describe("Dynamic Resource", func() {
-	It("resolve prometheus API", func() {
+var _ = FDescribe("Dynamic Resource", func() {
 
-		defer cleaner.NotifyTestComplete(GinkgoT(), true)
+	defer GinkgoRecover()
```

Comment:
`GinkgoRecover` is only necessary when a test spawns its own goroutine, so it's not needed here.

Reply:
The Ginkgo's doc says, 
if your test launches a goroutine that calls Fail (or, equivalently, invokes a **failing Gomega assertion),** there’s no way for Ginkgo to rescue the panic that Fail invokes. 
So, my understanding was every Gomega assertion that fails causes a panic and hence `GinkgoRecover()` is needed

Reply:
That should be read as "if your test launches a gorouting that calls Fail" OR "if your test launches a goroutine that invokes a failing Gomega assertion". They are both essentially the same thing, because [Gomega is configured to call Fail when an assertion fails](https://github.com/operator-framework/operator-lifecycle-manager/blob/bc06046b43f3f113a7c9e69b89cc3acc72a718bd/test/e2e/e2e_test.go#L43).